recipes-security/refpolicy/refpolicy-mcs_2.%.bbappend

```diff
@@ -202,7 +202,7 @@ POLICY_MLS_CATS = "256"
 EXTRA_OEMAKE += "PKGNAME=${POLICY_NAME}-${PV}"
 
 do_srctree_copy() {
-        cp -r ${WORKDIR}/policy ${S}/
+        cp -nr "${WORKDIR}/policy/modules" "${S}/policy"
```

Comment:
I'm concerned about the no-clobber option.  If we change a module in the layer, and bitbake triggers a re-build, will no-clobber result in the old, stale file still being in `${S}`?

Reply:
Good point, I was afraid of things in the layer overwriting upstream files, but the former is worse.
I think we can leave it up to the developers if they want to overwrite policy files from the layer or not. Additional logic to handle only overwriting files provided in the `policy` directory from the layer would not be worth it.